Description: Do not require Internet access during building.
 intersphinx want to fetch some mapping data, and was set up to use
 the Internet to find it.  Use a local file from python-doc instead to
 avoid violating Debian policy 4.9.
Author: Petter Reinholdtsen <pere@debian.org>
Bug-Debian: https://bugs.debian.org/830186
Forwarded: no
Reviewed-By: Petter Reinholdtsen <pere@debian.org>
Last-Update: 2016-08-27

--- python-requests-toolbelt-0.7.0.orig/docs/conf.py
+++ python-requests-toolbelt-0.7.0/docs/conf.py
@@ -268,4 +268,4 @@ texinfo_documents = [
 
 
 # Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'http://docs.python.org/': None}
+intersphinx_mapping = {'/usr/share/doc/python-doc/html': None}
